.circleci/config.yml

```diff
@@ -48,6 +51,9 @@ commands:
             ROUTE_SERVICE_NAME=waf-route-${DAPP_SPACE}-usagov \
             PROTECTED_APP_NAME=$DCMS_APP \
             ./bin/cloudgov/deploy-waf waf-<< pipeline.number >> $WAF_DIGEST
+            cf set-env waf USPS_USERID $USPS_USERID
```

Comment:
Do we really need these on the WAF? Just curious.

Reply:
Also, I removed those lines, but when I tested it on dev, it didn't work on beta-dev.usa.gov.

Reply:
Now I wonder whether it didn't work due to "maintenance mode." It's not doing any harm, though.

Reply:
Does maintenance mode only affect the static site? Because it was working in the cms.

Reply:
What it actually affects is anonymous access -- so if the site is in maintenance mode, the cms will serve different content if you're not logged in. The way this affects the static site is that the static site generator accesses the cms as an anonymous user ... and winds up not pushing content to the static site. So I guess the way it would have an effect here would be if your changes never made it to the static site.